### ISSUE TITLE
Fizz v4.5.10 type issue

### ISSUE THREAD

**Issue by debudda** [gobuffalo/pop#138]
Hi everyone.
I ran against an issue with fizz after upgrade to v4.5.10:
`Error: error processing migrations/booms.up.fizz: could not fizz the migration migrations/booms.up.fizz: parse error: function run error: function wants argument type fizz.Options but received type map[string]interface {}`
Had to add a constraint to resolve it.

**Comment by u007** [comment]
i think u may need to go get -u github.com/gobuffalo/fizz 
and also go get -u to update pop

**Comment by flexlee** [comment]
I ran into the same issue. `go get -u github.com/gobuffalo/fizz` doesn't help.

**Comment by mclark4386** [comment]
This is a known bug that is caused by the anko package changing. We are in the process of moving to plush so this doesn't happen again, but in the mean time if you are using `dep` you can lock anko to `0.0.4`

**Comment by stanislas-m** [comment]
`go get -u github.com/gobuffalo/fizz` won't help, since the concened branch is not merged yet. go get -u github.com/gobuffalo/pop`should work, though.

**Comment by flexlee** [comment]
@mclark4386  isn't current master of pop already locked anko at v0.0.4? Sorry I'm a bit confused.

Gopkg.lock
```
[[projects]]
  name = "github.com/mattn/anko"
  packages = [
    "ast",
    "core",
    "parser",
    "vm"
  ]
  revision = "e9c5c0ca86cf129292c56ddbddc4fff027844d65"
  version = "v0.0.4"
```

**Comment by markbates** [comment]
Update: The fix for this is here https://github.com/gobuffalo/pop/pull/130

**Comment by markbates** [comment]
fixed with the release of pop `v4.5.11` https://github.com/gobuffalo/pop/releases/tag/v4.5.11. Update your dependencies and you'll be good to go.

**Comment by markbates** [comment]
No need to pin `anko` any more, as it's been removed entirely! :)
